### PR TITLE
Update destinations.yml.j2 for pulsar_cz02

### PR DESCRIPTION
**Pull request by martindemko**
fixes alphafold db path for version 2.2

### CODE REVIEW

File: files/galaxy/tpv/destinations.yml.j2 [hunk 1]
```diff
@@ -287,7 +287,7 @@ destinations:
         env:
           ALPHAFOLD_DB: "/storage/brno11-elixir/projects/alphafold/alphafold.db-2.2"
         params:
-          singularity_volumes: "$job_directory:ro,$tool_directory:ro,$job_directory/outputs:rw,$working_directory:rw,/cvmfs/data.galaxyproject.org:ro,$SCRATCHDIR,/storage/praha5-elixir/home/galaxyeu:/home/galaxyeu,$ALPHAFOLD_DB:/data/2.2:ro"
+          singularity_volumes: "$job_directory:ro,$tool_directory:ro,$job_directory/outputs:rw,$working_directory:rw,/cvmfs/data.galaxyproject.org:ro,$SCRATCHDIR,/storage/praha5-elixir/home/galaxyeu:/home/galaxyeu,$ALPHAFOLD_DB:data/db/databases/alphafold_databases/2.2:ro"
```
Comment (bgruening) [inline files/galaxy/tpv/destinations.yml.j2:290]: Isn't there a `/` missing before data?

Comment (martindemko) [reply files/galaxy/tpv/destinations.yml.j2:290]: Sure it is. I'm too tired obviously

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Sleep well and have a nice weekend :)

**Comment by martindemko** [comment]
Thanks, you too :)

**Pull request merged**